components/dialpad/sources/new-call-event/new-call-event.ts

```diff
@@ -0,0 +1,27 @@
+import { defineSource } from "@pipedream/types";
+import constants from "../../common/constants";
+import common from "../../common/common";
+
+export default defineSource({
+  ...common,
+  type: "source",
+  name: "New Call Event",
+  key: "dialpad-new-call-event",
+  description: `Emit new call event subscription. 
+  See [Event doc](https://developers.dialpad.com/reference/webhook_call_event_subscriptioncreate)
+  and [webhook doc](https://developers.dialpad.com/reference/webhookscreate)`,
+  version: "0.0.1",
+  methods: {
```

Comment:
We should use `dedupe` just in case duplicate events come through. This applies to the rest of the sources as well.
```suggestion
  version: "0.0.1",
  dedupe: "unique",
  methods: {
```